config/common/activegates/serviceaccount-activegates.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: dynatrace-activegates
```

Comment:
Wouldn't `dynatrace-activegate` fit better (similar to the other Service Accounts)?

Reply:
but this can have multiple activegates inside :)

Reply:
isn't it still one activegate that has multiple capabilities?

Reply:
😞 I hate naming things